pkg/controller/healthz.go

```diff
@@ -0,0 +1,34 @@
+package controller
+
+import (
+	"net/http"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/logging"
+	"github.com/google/exposure-notifications-verification-server/pkg/render"
+)
+
+func HandleHealthz(h *render.Renderer, cfg *database.Config) http.Handler {
```

Comment:
I'm worried that having this open and unauthenticated and not rate limited makes it easy to burn through available SQL connections (since this creates a new connection each time)

Reply:
That's reasonable. We can totally put behind the rate limiter, or just remove the db portion. Whichever you prefer.

Reply:
random options

- put a mutex in this handler + only actually run the check ever 30/60 seconds
- drop the connection open and just do the ping
- put in IP based rate limiting w/ a pretty low limit
- make this IP whitelist or require an API key?

Reply:
I'd like to learn more about why we need /healthz endpoints? AFAIK, Cloud Run can't poke them. If we're on K8S, I think the health endpoints should bind to 127.0.0.1 instead of 0.0.0.0 so they are only available in-pod.

Reply:
Ping will always open a connection https://golang.org/pkg/database/sql/#DB.Ping, but a mutex is easy enough. Added time/rate to do this.

Reply:
@sethvargo this is to allow stackdriver (or any other prober) to hit our service with as little business logic is possible. This isn't for our deploy system to do a check but rather for external black box probing. It's especially needed since the majority of our systems don't have a consistent url that always returns a 200 on unauthed GET.